invenio_previewer/extensions/csv_papaparsejs.py

```diff
@@ -17,7 +17,14 @@
 
 def can_preview(file):
     """Determine if the given file can be previewed."""
-    return file.is_local() and file.has_extensions(".csv", ".dsv")
+    max_file_size = current_app.config.get(
```

Comment:
Would it make sense to have a validator function (i.e. `validate_csv`) to separate concerns? That way, it would be easier to extend in the future in case more rules are needed.

This is done for other extensions (e.g. [json](https://github.com/inveniosoftware/invenio-previewer/blob/141038234f0d7a64a28cc651b3b65c8ff58eef74/invenio_previewer/extensions/json_prismjs.py))